lib/resolvers/PackageResolver.ts

```diff
@@ -105,6 +117,24 @@ export class PackageResolver {
     });
   }
 
+  /**
+   * Retrieves the latest Angular Material version remotely.
+   */
+  private static _retrieveLatestVersion(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      request('http://material.angularjs.org/docs.json', (error, response, body) => {
```

Comment:
That feels a little hacky. I'll take a look around to see what's available, [npm-package-registry](https://www.npmjs.com/package/npm-registry-client) seems like it could get the job done.